modules/vaos/config/routes.rb

```diff
@@ -5,6 +5,7 @@
     resources :appointments, only: %i[index create] do
       put 'cancel', on: :collection
     end
+    get '/appointments/:type/:id', to: 'appointments#show', type: /va/
```

Comment:
we can limit the REXEX with `/\Ava\z/` so that it doesn't match on vaos for example.

Reply:
We tried adding the `\A\z` anchors to the regex and Ruby didn't like that.